### ISSUE TITLE
toolchain.py fails with Xcode 9.0

### ISSUE THREAD

**Issue by sastels** [kivy/kivy-ios#265]
Running
```
toolchain.py build python
```
fails with the newly released Xcode 9.0
```
/Modules/posixmodule.c:2661:11: error: 'system' is unavailable: not available
      on iOS
    sts = system(command);
          ^
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.0.sdk/usr/include/stdlib.h:195:6: note:
      'system' has been explicitly marked unavailable here
int      system(const char *) __DARWIN_ALIAS_C(system);
```

full log: [log.txt](https://github.com/kivy/kivy-ios/files/1321463/log.txt)

Note: Similar to issues with beta Xcode mentioned in #258

**Comment by fermanjj** [comment]
Also having this same issue.

MacOS High Sierra Version 10.13
Xcode Version 9.0

**Comment by tonibagur** [comment]
I had the same problem and I managed to fix it, I submited the following pull request:
https://github.com/kivy/kivy-ios/pull/268
I hope that this fix help you too.

**Comment by mbrockman1** [comment]
This is still failing for me when trying to run '/toolchain.py build kivy'. Here is the total [output](https://gist.githubusercontent.com/mbrockman1/7cb7a11a71d5d73cf7b9e78e6b9d78b0/raw/1bd8147f6c774975532b44dbc1b7cdc1e7d63b5e/gistfile1.txt)

**Comment by bgriffen** [comment]
I used the latest clone and still found same error. Using Sierra 10.12.6 and XCode 9.1 (9B55).

**Comment by akshayaurora** [comment]
closed by #268
